packages/core/src/tools/schema.js

```diff
@@ -54,21 +55,26 @@ const convertResourceDos = (appRaw) => {
     }
 
     if (search && create && isVisible(search) && isVisible(create)) {
-      const searchOrCreate = {
-        // key: `${resource.key}SearchOrCreate`,
+      // searchAndCreates is an alias for searchOrCreates. Schema validation makes sure only one of them is defined.
+      const searchCreates = appRaw.searchAndCreates
+        ? appRaw.searchAndCreates
+        : appRaw.searchOrCreates;
```

Comment:
There are two unused variables, `searchOrCreates` and `searchAndCreates`, in the beginning of this function. I think they probably should be used here instead?